config/job-config.yaml

```diff
@@ -2,7 +2,7 @@
 # The tools will work out all permutations of runs from this.
 # They will interrogate the database and see what runs already exist.
 # Any runs that need running, will be.
-
+jobId: Random_Job_ID
```

Comment:
Let's:

- Find a broader top-level for this.  We will probably eventually need to have a few things passed from FaaS to jenkins-sdk.  Let's not have loads of new top-level keys for that.
- Probably leave it out of this particular file.  It's just confusing and users won't know whether they are meant to supply something - plus all daily FIT/PERF runs will get put in the same job?  It can just be added only by FaaS.

Reply:
What does this look like now  - is it?

```
jc:
  jobId: 
```

Can we use something more descriptive and intuitive than `jc` (is that job config?).  How about:

```
job:
  jobId:
```